src/lib.rs

```diff
@@ -367,6 +368,16 @@ pub struct FlamegraphOptions {
     /// Image width in pixels
     #[structopt(long = "image-width")]
     pub image_width: Option<usize>,
+
+    /// Color palette
+    #[structopt(
+        long = "palette",
+        possible_values = &[
```

Comment:
What's the value of listing all possible values here? A downside is that we have to keep this list in sync with inferno.

Reply:
That's a good question, and TBH I listed them simply because they're available, thinking it would be nice to have as many options as possible for the `flamegraph` and `cargo-flamegraph` binaries.  There are some general reasons for having multiple palettes available, including:
* choice in case some palettes are more or less readable for a particular end-user
* profiling an app under different parameters and using separately colored flamegraphs for comparison

 I also thought that there was a way to automatically generate the list of palettes (using `clap::arg_enum!`), but I think that would have to be implemented in the inferno repo.

I guess I don't have the full picture of how this tool is used, but it appears to be the main cli frontend to the inferno lib, and would therefore want to harness the full capabilities of the backend.

As far as maintainability, I suppose we would still need to manually update the `possible_values` any time the palettes change in inferno, whether or not we use the complete list.  For example, I have a pending PR for adding `--palette rust` that we're adding here, and I could see other variants added down the line.  Somehow I don't imagine variants being removed, and I suspect new variants would be rare.

Hopefully that all makes sense, and I'm looking forward to hearing if you have suggestions on how to proceed.  Thanks!

Reply:
I think you don't have to make the possible values explicit because the FromStr impl for Palette will take care of validation.

Reply:
Yes the validation is indeed handled by the FromStr impl, and I tested it out over here just to make sure.  A better answer to your question is that using `possible_values` will display a list of options to the user when running `cargo flamegraph -h`.  So the value is in documentation and discovery.